kubeprod/pkg/eks/platform.go

```diff
@@ -305,6 +426,9 @@ func (conf *Config) Generate(ctx context.Context) error {
 			return err
 		}
 		conf.ContactEmail = email
+		if conf.ContactEmail == "" {
```

Comment:
Hrm - we don't do this check in other platforms.  Is it required? (does cert-manager fail to function with an empty-string email address?) 
If it is required, we should probably mark the flag as required instead of this check (and do the same across the other platforms).

Reply:
Yes, cert-manager will fail because the letsencrypt-prod clusterissuer will get upset if an e-mail is not provided.

Reply:
I'll disable the requirement for now, and will fix this in a separate PR (to make the e-mail mandatory).